test/integration/stop/test_stop.py

```diff
@@ -0,0 +1,90 @@
+import os
+import shutil
+import subprocess
+
+from utils import pid_file, run_path, wait_file
+
+
+def app_cmd(tt_cmd, tmpdir_with_cfg, cmd, input):
+    start_cmd = [tt_cmd, *cmd]
+    tt_process = subprocess.Popen(
+        start_cmd,
+        cwd=tmpdir_with_cfg,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        stdin=subprocess.PIPE,
+        text=True
+    )
+
+    tt_process.stdin.writelines(input)
+    tt_process.stdin.close()
+    rc = tt_process.wait()
+    assert rc == 0
+    return tt_process.stdout.readlines()
+
+
+def test_stop(tt_cmd, tmpdir_with_cfg):
+    shutil.copy(os.path.join(os.path.dirname(__file__), "test_app.lua"), tmpdir_with_cfg)
+    app_name = "test_app"
+    start_output = app_cmd(tt_cmd, tmpdir_with_cfg, ["start", app_name], [])
+    assert "Starting an instance" in start_output[0]
+    assert wait_file(os.path.join(tmpdir_with_cfg, app_name, run_path, app_name),
+                     pid_file, []) != ""
+
+    try:
+        # Test confirmed stop.
+        stop_output = app_cmd(tt_cmd, tmpdir_with_cfg, ["stop", app_name], ["y\n"])
+        assert "Confirm stop of 'test_app' [y/n]" in stop_output[0]
+        assert "has been terminated" in stop_output[0]
+
+        # Test cancelled stop.
+        stop_output = app_cmd(tt_cmd, tmpdir_with_cfg, ["stop", app_name], ["n\n"])
+        assert "Confirm stop of 'test_app' [y/n]" in stop_output[0]
```

Comment:
It is rather unusual that we are requesting confirmation to stop an already stopped process, is it not?

Reply:
Yes it is, added check for file like you suggested